apps/script/generateSharedConstants.rb

```diff
@@ -96,8 +96,9 @@ def main
     CHILD_ACCOUNT_COMPLIANCE_STATES
     CENSUS_CONSTANTS
     DANCE_SONG_MANIFEST_FILENAME
-    AI_TUTOR_INTERACTION_SAVE_STATUS
+    AI_TUTOR_INTERACTION_STATUS
```

Comment:
See the rationale below for refactoring here. Is it dangerous to refactor shared constants?

Reply:
Seems ok to me, especially since it's something pretty clearly owned/consumed by your team?

Reply:
Thinking about Ben's comment at https://github.com/code-dot-org/code-dot-org/pull/57669/files#r1548141051 - are their plans to use `STATUS` on the backend for `aichat` @sanchitmalhotra126 ?

Reply:
I think we could but it's a bit unclear since our API design is still being finalized. I'm okay with keeping this in the AI Tutor space for now and revisiting/refactoring when the API is more solidified.

Reply:
When I said "dangerous," what I was thinking of was, "Is there some script or something I need to reseed this constant file?" If tests are passing, I'll assume not. I'm happy to refactor this more down the line if it's useful for generative AI.